docs/robots.txt

```diff
@@ -0,0 +1,5 @@
+User-agent: *
```

Comment:
are you sure this file is served correctly, without adding it to `mkdocs.yml`?

Reply:
yes, it's available in the PR preview of the docs, should be correct this way